<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>585</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="des_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="train_text">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Select Label Class</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="classes_cBox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Add Label Class</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="class_editText"/>
        </item>
        <item>
         <widget class="QPushButton" name="addClass_bt">
          <property name="text">
           <string>Add Label</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="saveTrain_bt">
         <property name="text">
          <string>Save Train</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveModel_bt">
         <property name="text">
          <string>save Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrain">
    <property name="title">
     <string>Train</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionOpenModel"/>
    <addaction name="actionSaveModel"/>
   </widget>
   <widget class="QMenu" name="menuCam">
    <property name="title">
     <string>Train</string>
    </property>
    <addaction name="actionStartTrain"/>
    <addaction name="actionStopTrain"/>
   </widget>
   <widget class="QMenu" name="menuRecognition">
    <property name="title">
     <string>Recognition</string>
    </property>
    <addaction name="actionStartRecognition"/>
    <addaction name="actionStopRecognition"/>
   </widget>
   <addaction name="menuTrain"/>
   <addaction name="menuModel"/>
   <addaction name="menuCam"/>
   <addaction name="menuRecognition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addTrain"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpenModel">
   <property name="text">
    <string>Open Model</string>
   </property>
  </action>
  <action name="actionSaveModel">
   <property name="text">
    <string>Save Model</string>
   </property>
  </action>
  <action name="actionStartTrain">
   <property name="text">
    <string>Start Train</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T, Ctrl+A</string>
   </property>
  </action>
  <action name="actionStopTrain">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Train</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T, Ctrl+Q</string>
   </property>
  </action>
  <action name="addTrain">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/addTrain.png</normaloff>:/images/addTrain.png</iconset>
   </property>
   <property name="text">
    <string>Add Train</string>
   </property>
  </action>
  <action name="actionStartRecognition">
   <property name="text">
    <string>Start recognition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, Ctrl+A</string>
   </property>
  </action>
  <action name="actionStopRecognition">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop recognition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
